dev/com.ibm.ws.jsp.2.3_fat/publish/servers/jstlServer/server.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+    Copyright (c) 2021 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Server for testing JavaServer Pages 2.3">
+
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+        <feature>jsp-2.3</feature>
+    </featureManager>
+
+    <javaPermission className="java.util.PropertyPermission" name="dtm.debug" actions="read"/> user.dir
+
+     <javaPermission className="java.util.PropertyPermission" name="user.dir" actions="read"/>
```

Comment:
I made this server instead of using the jspServer because of these permissions. They were encountered when using the xml tags.